src/App.tsx

```diff
@@ -10,31 +12,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    setSelectedTab(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
 
       <div data-cy="TabsComponent">
         <div className="tabs is-boxed">
```

Comment:
You have these wrappers in the `Tabs` component.  It doesn't look good. I think you can remove it
```suggestion
```
![image](https://user-images.githubusercontent.com/99403198/206855028-09213d86-4303-43c2-8ae0-e9a7b14aa254.png)